components/tab/IconTab.js

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import {
+  IconCalendarMediumOutline,
+  IconCheckmarkMediumOutline,
+  IconContactsMediumOutline,
+  IconInvoiceMediumOutline,
+  IconProductsMediumOutline,
+  IconStatsMediumOutline,
+  IconTimerMediumOutline,
+} from '@teamleader/ui-icons';
+
+const iconMap = {
+  crm: IconContactsMediumOutline,
+  invoices: IconInvoiceMediumOutline,
+  products: IconProductsMediumOutline,
+  stats: IconStatsMediumOutline,
+  time: IconTimerMediumOutline,
+  deals: IconCheckmarkMediumOutline,
+  planning: IconCalendarMediumOutline,
+};
+
+class IconTab extends PureComponent {
+  static propTypes = {
+    active: PropTypes.bool,
+    children: PropTypes.node.isRequired,
+    className: PropTypes.string,
+    counter: PropTypes.node,
+    element: PropTypes.node,
+    href: PropTypes.string.isRequired,
```

Comment:
`href` should not be listed in the propTypes because you can now also use other elements.